### ISSUE TITLE
Unresolved warnings on build

### ISSUE THREAD

**Issue by pyrex41** [ricobank/snek#10]
Using Apple M1 Max.

`npm run initialize` runs / completes fine (once I initialized the ipfs daemon -- that wasn't mentioned in Readme but dug through the error messages)

I get the following error when I run `npm run build-general`:
<img width="739" alt="Screen Shot 2022-07-12 at 5 23 43 PM" src="https://user-images.githubusercontent.com/12162406/178622847-6e776e7e-139f-4739-8fb5-097c4023b961.png">

**Comment by pyrex41** [comment]
should note that I am able to use snek just fine by just aliasing `snek` to `node ~/snek/index.js $argv`

**Comment by dmfxyz** [comment]
Can you confirm that `npm run build-general` doesn't produce a usable `snek` binary in the same directory as the repo? I get similar warnings on mac but still get snek built. 

these warnings should be looked at, just want to understand the issue.

**Comment by nmushegian** [comment]
I think building a single binary like this is pretty non-standard, wondering if we can't just go with `npm i -g` type of thing. @stobiewan thoughts

**Comment by pyrex41** [comment]
@dmfxyz you are correct, the binary actually does build; apparently I missed / made assumptions based on the errors. Updated issue title accordingly.

**Comment by stobiewan** [comment]
Originally added "build-linux" script just because it's helpful for testing and using myself. Something like npm install -g would be ideal, assume vyper+python devs are OK with using npm to get snek.
